tools/stub_generator/stub_codegen.py

```diff
@@ -25,7 +25,17 @@
 
 
 def function_header(return_type, name, args):
-    arg_str = ", ".join([f"{arg_type} {arg_name}" for arg_type, arg_name in args])
+    args_expr = []
+    for arg_type, arg_name in args:
+        # handle array types differently, put size after the name
+        if "[" in arg_type:
+            pos = arg_type.find("[")
+            type = arg_type[0:pos]
+            args_expr.append(f"{type} {arg_name}{arg_type[pos:]}")
+        else:
+            args_expr.append(f"{arg_type} {arg_name}")
```

Comment:
It should be before first `[` or before first `)`:
| Type         | Declaration     | Comment                    |
|---------------|-----------------|----------------------------|
| `int[]`       | `int a[]`       | array                      |
| `int (*)()`   | `int (*a)()`    | function pointer           |
| `int (*[])()` | `int (*a[5])()` | array of function pointers |
| `int (&)[5]`  | `int (&a)[5]`   | reference to an array      |

Reply:
Fixed (I hope)